jobs/transforms/exp_validation.py

```diff
@@ -45,3 +46,47 @@ def _get_change_domain_class(self, input_model): # pylint: disable=unused-argume
             changes made by commit commands of the model.
         """
         return exp_domain.ExplorationChange
+
+
+@validation_decorators.AuditsExisting(
+    exp_models.ExplorationRightsSnapshotMetadataModel)
+class ValidateExplorationRightsSnapshotMetadataModel(
+        base_validation.BaseValidateCommitCmdsSchema):
+    """Overrides _get_change_domain_class for exploration models """
+
+    def _get_change_domain_class(self, input_model): # pylint: disable=unused-argument
+        """Returns a Change domain class.
+
+        Args:
+            input_model: datastore_services.Model. Entity to validate.
+
+        Returns:
+            change_domain.BaseChange. A domain object class for the
+            changes made by commit commands of the model.
+        """
+        return rights_domain.ExplorationRightsChange
+
+
+@validation_decorators.AuditsExisting(
+    exp_models.ExplorationCommitLogEntryModel)
+class ValidateExplorationCommitLogEntryModel(
+        base_validation.BaseValidateCommitCmdsSchema):
+    """Overrides _get_change_domain_class for exploration models """
+
+    def _get_change_domain_class(self, input_model):
+        """Returns a Change domain class.
+
+        Args:
+            input_model: datastore_services.Model. Entity to validate.
+
+        Returns:
+            change_domain.BaseChange. A domain object class for the
+            changes made by commit commands of the model.
```

Comment:
```suggestion
            ExplorationRightsChange|ExplorationChange. A domain object class for the
            changes made by commit commands of the model.
```

Reply:
Done

Reply:
@vojtechjelinek this time done for sure PTAL!